### DISCUSSION TITLE
Jupyter notebooks (.ipynb) don't support dark mode

### DISCUSSION THREAD

**Discussion by oxygen-dioxide** [community/community#3090] [Other Feature Feedback, Questions, & Ideas] [answered]
Jupyter notebooks (.ipynb) don't support dark mode. For example, https://github.com/NELSONZHAO/zhihu/blob/master/mnist_gan/gan_mnist.ipynb
![image](https://user-images.githubusercontent.com/54425948/103406871-32d69280-4b97-11eb-95cb-50dfe032db01.png)

**Comment by evi-liu** [comment, marked as answer]
This space is currently reserved for feedback on Discussions and GitHub for Mobile. This is noted in our [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).
